TFT/src/User/API/interfaceCmd.c

```diff
@@ -78,41 +78,6 @@ bool storeCmd(const char * format, ...)
   return true;
 }
 
-// Parse and store gcode command script to infoCmd queue.
```

Comment:
storeScript should be removed also from interfaceCmd.h

Reply:
Yes, it must go. Thanks!